lib/engine/game/base.rb

```diff
@@ -1950,9 +1950,9 @@ def remove_train(train)
       end
 
       def rust(train)
+        train.rusted = true
         remove_train(train)
         train.owner = nil
-        train.rusted = true
```

Comment:
This is probably okay. Did you check all games that implement remove_train to verify this change doesn't mess them up?

Reply:
I looked at '28 and NY which seem to be the only games that shadow this function, and `rusted` doesn't seem to be a flag that they are checking or influencing their logic

Reply:
OK with keeping this change to base code even if the bugfix implementation doesn't require it?

Reply:
remove_train is the method in question. It is overridden by 17, 22, 28, 77, and NY.

Reply:
Ah yes I checked 22 as well which seemed to be fine

'17 is this one so it should work

77 I think is a different function? `def remove_trainless`